code/modules/hydroponics/seeds.dm

```diff
@@ -20,6 +20,33 @@
 	GLOB.seed_list -= src
 	return ..()
 
+/obj/item/seeds/verb/rename_seeds() //set amount_per_transfer_from_this
+	set name = "Rename Seeds"
+	set category = "Object"
+	set src in usr
+	if(!ishuman(usr))
+		return
+	var/mob/living/carbon/human/user = usr
+	var/obj/item/seeds/S = user.get_active_hand()
```

Comment:
not sure about forcing it to be in their active hand, either hand it to be on their person or adjacent them